testing/singularity/APKBUILD

```diff
@@ -0,0 +1,76 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=singularity
+pkgver=3.1.1
+pkgrel=0
+pkgdesc="Application containers focused on reproducibility for scientific computing and HPC world."
+url="https://www.sylabs.io/singularity/"
+arch="all"
+license="BSD-3-Clause BSD-3-Clause-LBNL"
+options="suid"
+#depends="squashfs-tools"
```

Comment:
Why is this commented out?

Reply:
@Ikke Installation instruction (https://github.com/sylabs/singularity/blob/master/INSTALL.md) suggest to install several packages on Ubuntu. I had my doubts about some of them, in particular about `libgpgme11-dev` `pkg-config` `squashfs-tools`, so I was moving them from `makedepends` to `#depends` to check the build.

Reply:
@Ikke I have enabled `depends="squashfs-tools"` as `squashfs-tools` are used during `singularity build` stage.